### DISCUSSION TITLE
Actual Budget LXC deployment error

### DISCUSSION THREAD

**Discussion by dtom2444** [tteck/Proxmox#2751] [Help Needed]
Tried deploying the Actual Budget LXC using the bash command and when running the install with verbose logging, I see the error:
-➤ YN0000: · Yarn 4.0.1
\➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed in 0s 596ms
➤ YN0000: ┌ Post-resolution validation
➤ YN0002: │ actual-sync@workspace:. doesn't provide @babel/core (pd13b1), requested by @babel/preset-typescript.
➤ YN0086: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code.
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0000: ⠙ ===============================================================================--
|
[ERROR] in line 39: exit code 0: while executing command $STD yarn install

Anyone know how to resolve this?
Thanks.

![image](https://github.com/tteck/Proxmox/assets/50755875/5443a193-cec2-400c-9e19-ae97fbddd6de)

**Comment by dtom2444** [comment]
Here is the error when using default settings:
 ✓ Installed Node.js
 - Installing Actual Budget  |
[ERROR] in line 23: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

![image](https://github.com/tteck/Proxmox/assets/50755875/84bc6cae-0b12-42fb-b8c1-089cf0e7b3d0)

**Comment by tteck** [comment]
🏁 The script runs without any problems from this end.

```
    ___        __              __   ____            __           __
   /   | _____/ /___  ______ _/ /  / __ )__  ______/ /___ ____  / /_
  / /| |/ ___/ __/ / / / __ `/ /  / __  / / / / __  / __ `/ _ \/ __/
 / ___ / /__/ /_/ /_/ / /_/ / /  / /_/ / /_/ / /_/ / /_/ /  __/ /_
/_/  |_\___/\__/\__,_/\__,_/_/  /_____/\__,_/\__,_/\__, /\___/\__/
                                                  /____/
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 110
Using Hostname: actualbudget
Using Disk Size: 4
Allocated Cores: 1
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Actual Budget LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 110 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.114.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Installed Actual Budget
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Actual Budget should be reachable by going to the following URL.
         http://192.168.86.250:5006 
```

**Comment by papadopouloskyriakos** [comment]
Same issue here, 1st tried with custom settings and failed as well 
![image](https://github.com/tteck/Proxmox/assets/37277364/c11d9bd9-1d5d-4d09-b05b-d3b54e3bd01d)

**Reply by tteck** [reply]
🔁 If I cannot reproduce the problem on my side, I cannot address an issue that appears to be functioning correctly here.

Try a different storage. 🤷🏻‍♂️

**Comment by tteck** [comment]
Try increasing resources.

**Reply by papadopouloskyriakos** [reply]
yeap, that did the trick! x1000 thanks!

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/commit/9c19734b5b3e0910bc2fa3dc60867251003b58bd
